webhook/resourcesemantics/validation/reconcile_config.go

```diff
@@ -143,6 +144,29 @@ func (ac *reconciler) reconcileValidatingWebhook(ctx context.Context, caCert []b
 			},
 		})
 	}
+	for gvk, callback := range ac.callbacks {
```

Comment:
Is this related to the recent broker webhook issue that @aliok ran into?

Reply:
Yes